README.md

```diff
@@ -113,3 +151,42 @@ For support, please @ David Calder in the [volunteer-app](https://scottishtechar
 ## Known issues
 * The iOS simulator only works with the IP Address of the Load Balancer as the value of STA_BASE_URL:
    * `  STA_BASE_URL: 'http://18.134.220.155',`
+
+# App deployment
+
+**If you're on a Mac** you can deploy the Android and iOS versions of the app.  
+
+**If you're on Windows/Linux** you can only deploy the Android version of the app (you'll always need to get another team member with a Mac to deploy the iOS version).
+
+1. In the pull request for the changes you're making (e.g. a new app feature), before you submit the PR for review, update the `version` number in `app/package.json`.  Normally for minor features/fixes, just update the last part of the version number (e.g. `"1.0.24"` becomes `"1.0.25"`).
+
+2. Update `/app/android/app/build.gradle` at the same time *(be careful here! there are other files called `build.gradle` in other similar directories)*.  About halfway down the file there are two things you need to update:
+
+    a. `versionCode` - this must be 1 higher than the existing number, e.g. `48` becomes `49` (don't use any dots in this one), no quote marks
```

Comment:
Not for now, but this is something you can automate with Fastlane – it looks at the latest version in Play Store / App Store and bumps it by one number automatically. We could maybe add a ticket in the backlog for doing this

Reply:
Ah that's cool, I didn't realise that.  I'll add a ticket on this for later.

Reply:
https://sta2020.atlassian.net/browse/SVA-211

We could update the app version number displayed in the app at the same time?

Reply:
Yeah, we should be able to pull both the app `versionCode` and `versionName` using [this package](https://www.npmjs.com/package/react-native-version-number) and display them in that exampleContainer or  somewhere else in the app for debugging 🎉

Reply:
Ah brilliant, thanks -- love a time-saver!  I've added the link to that package to that Jira ticket I created, for whoever picks that up 👍